src/components/landing/LandingPageOneToOne.module.css

```
@@ -1156,16 +1254,80 @@
+
+  .heroVisual {
+    width: min(100%, 36rem);
+    margin-top: 0;
+    margin-inline: auto;
+    overflow: visible;
+  }
+
+  .heroVisualStage {
+    aspect-ratio: 1.44;
+    --hero-desktop-right: 4%;
+    display: flex;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)